lib/browser/api/auto-updater/squirrel-update-win.js

```diff
@@ -17,8 +17,8 @@ const isSameArgs = (args) => args.length === spawnedArgs.length && args.every((e
 
 // Spawn a command and invoke the callback when it completes with an error
 // and the output from standard out.
-const spawnUpdate = function (args, detached, callback) {
-  let error, errorEmitted, stderr, stdout
+const spawnUpdate = function(args, detached, callback) {
+  let error; let errorEmitted; let stderr; let stdout
```

Comment:
This change is surprising; multiple `let` seems ok to me? is it a google style guide thing? is there something bad about multiple `let`?

Reply:
@nornagon This is a result of running `lint:js --fix`, this is a result of the [`one-var`](https://eslint.org/docs/rules/one-var) rule.  Set here in [google's eslint config](https://github.com/google/eslint-config-google/blob/master/index.js#L277-L281)

Reply:
The autofixed code is worse though.

Reply:
I think this autofixed code could be improved by moving the `let` declarations to the point at which the variables are actually used. I'm OK with this lint rule staying as is.